testnet/stacks-node/src/tests/epoch_25.rs

```diff
@@ -169,17 +170,25 @@ fn microblocks_disabled() {
     submit_tx(&http_origin, &tx);
 
     // wait until just before epoch 2.5
-    loop {
+    wait_for(30, || {
         let tip_info = get_chain_info(&conf);
-        if tip_info.burn_block_height >= epoch_2_5 - 2 {
-            break;
+        if tip_info.burn_block_height >= epoch_2_1 - 2 {
```

Comment:
Is this change (from 2.5 to 2.1) intentional? If so, can you update the comment above, and the `expect` below?